packages/dds/tree/package.json

```diff
@@ -102,11 +102,14 @@
 		"check:exports:bundle-release-tags": "api-extractor run --config api-extractor/api-extractor-lint-bundle.json",
 		"check:exports:cjs:alpha": "api-extractor run --config api-extractor/api-extractor-lint-alpha.cjs.json",
 		"check:exports:cjs:beta": "api-extractor run --config api-extractor/api-extractor-lint-beta.cjs.json",
-		"check:exports:cjs:legacy": "api-extractor run --config api-extractor/api-extractor-lint-legacy.cjs.json",
+		"check:exports:cjs:legacy-alpha": "api-extractor run --config api-extractor/api-extractor-lint-legacy-alpha.cjs.json",
```

Comment:
These `legacy-alpha` and `legacy-beta` checks are running against the `.legacy.d.ts` we generate from our `@legacy @beta` exports, so they're doing the same thing as the existing `legacy` checks. I don't think we should need to add these or the corresponding configs.

Reply:
The legacy-beta check is also redundant with the existing legacy check.

Reply:
Remove them too